src/dapla_metadata/variable_definitions/utils/variable_definition_files.py

```diff
@@ -204,11 +206,19 @@ def _populate_commented_map(
 ) -> None:
     """Add data to a CommentedMap."""
     commented_map[field_name] = value
-    description: JsonDict = cast(JsonDict, model_instance.model_fields[field_name].json_schema_extra[NORWEGIAN_DESCRIPTIONS])  # type: ignore[index]
+    field = model_instance.model_fields[field_name]
+    description: JsonDict = cast(JsonDict, field.json_schema_extra[NORWEGIAN_DESCRIPTIONS])  # type: ignore[index]
     if description is not None:
+        new_description = (
+            (OPTIONAL_FIELD + "\n" + str(description))
+            if not field.is_required()
+            else (REQUIRED_FIELD + "\n" + str(description))
+            if field.is_required()
+            else description
+        )
```

Comment:
```suggestion
        new_description = (REQUIRED_FIELD if field.is_required() else OPTIONAL_FIELD) + "\n" + str(description)
```

Isn't the last branch unreachable? Since `is_required()` returns a `bool`?

Reply:
yes, I would think so. I included it in case it was possible "is_required" could return "None"/not set. But you are right, i think it is always True/False